task/golang-test/0.3/golang-test.yaml

```diff
@@ -0,0 +1,135 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: golang-test
+  labels:
+    app.kubernetes.io/version: "0.3"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Testing
+    tekton.dev/tags: test
+    tekton.dev/displayName: "golang test"
+    tekton.dev/platforms: "linux/arm64,linux/amd64,linux/s390x,linux/ppc64le"
+spec:
+  description: >-
+    This is a Golang task to test Go projects.
+
+  params:
+  - name: package
+    description: package (and its children) under test
+  - name: packages
+    description: "packages to test (default: ./...)"
+    default: "./..."
+  - name: context
+    description: path to the directory to use as context.
+    default: "."
+  - name: image
+    description: docker image to run the task
+    default: "docker.io/library/golang"
+  - name: version
+    description: golang version to use for tests
+    default: "latest"
```

Comment:
instead of `latest` can we tag to a particular version so that in future tests don't fail?